Dockerfile

```diff
@@ -14,8 +14,9 @@ ENV GOPATH /go
 COPY --from=0 /go/ $GOPATH
 ENV PATH $PATH:$GOPATH/bin
 
-RUN apk add --update python python-dev py-pip build-base && \
-    pip install enum34 retrying google-api-core google-cloud-storage && \
+RUN apk add --update python3 python-dev py-pip build-base && \
+    ln -sf python3 /usr/bin/python && \
```

Comment:
I'm still torn about whether we should modify the python alias to use python3 or just explicitly use python3 at the line 42, as I can see arguments both ways.

No-op, just stating my concern - maybe Aaron will have a definitive preference one way or another...

Reply:
I don't see the downside to using python3 directly - what am I missing @tneymanov

Reply:
There are no behavioral downsides, only prophylactic ones - the only benefit I see is if someone in the future would modify the file, they could use 'python' command as is, without realizing that it's p2.

We just need to enforce one of the two behaviors everywhere:
 - always override python alias to be the appropriate version and use it everywhere; or
 - always explicitly use the python version you intend to use.

I'm ok with either one.

Reply:
So ideally we'd be explicit in our own usage.  I am not concerned about what happens to users that invoke the wrong version within our container, but I would be sad if we had a bug because *we* invoked the wrong thing.

So I think we actually want to modify the symlink (so we don't run 2 by accident) *and* maybe say python3 when we mean it.

WDYT?

Reply:
SGTM. I'm not quite clear on what cases would constitute explicit usage of python3, but I guess those special occurrences can be handled when they appear - shouldn't block the PR.